### ISSUE TITLE
Full support for contenteditable attribute

### ISSUE THREAD

**Issue by fanglingsu** [fanglingsu/vimb#248]
According to issue #237 there is no full support for the contenteditable attribute at the time.

- Allow to focus contenteditable elements by hinting `;o` and switch vimb ton input mode.
- Allow to use `;e` hinting to open the text content in the editor and to write the content back to the element.
- If the contenteditable element is still focused, allow `CTRL-T` to edit it's text content in editor.

**Comment by fanglingsu** [comment]
Special case of this is #396.

**Comment by AdrienLemaire** [comment]
@fanglingsu I'm unable to use the debugger console (gF then go to 'console' tab), some keys (eg 't') are interpreted like in normal mode

![console](https://user-images.githubusercontent.com/260983/73589023-02fcb600-4514-11ea-8b54-8db2463d510d.png)

Ctrl-T and ;e will open the external editor with the first input from the page, it doesn't seem possible to edit the debugger console input from the external editor.

Is there a fix or trick to be able to use the console?

**Comment by AdrienLemaire** [comment]
Solution: Detach into separate window

**Comment by fanglingsu** [comment]
@AdrienLemaire Yes you can detach. I gave up using keyboard when the developer tools are open. At the time the key events of the webview are processed - and since webkit2 the developer tools are a part of the webview. I think we could bypass the key processing logic if the developer tools are opened. But than we can not close them by pressing gF again or relaod the page by `r`. So it might be difficult to solve this issue.
